$OpenBSD: patch-orc_orcparse_c,v 1.1 2011/08/01 20:50:10 naddy Exp $

Make sure the log buffer starts out with a valid zero-length string.

--- orc/orcparse.c.orig	Mon Aug  1 16:33:42 2011
+++ orc/orcparse.c	Mon Aug  1 16:35:04 2011
@@ -71,6 +71,7 @@ orc_parse_full (const char *code, OrcProgram ***progra
   parser->log = malloc(100);
   parser->log_alloc = 100;
   parser->log_size = 0;
+  parser->log[0] = 0;
 
   while (parser->p[0] != 0) {
     char *p;
